content/tutorials/portenta-h7/por-ard-usb/content.md

```diff
@@ -23,12 +28,14 @@ It is possible to configure Portenta H7 to act as a USB host in a way that we ca
 -   USB cable compatible with the Arduino Arduino MKR WiFi 1010 (or the board you have selected)
 -   3 Jumper wires
 
-![The Arduino core is built on top of the Mbed stack](assets/Arduino-Logo.svg)
+# USB Host and Device mode 
+
+As we know, Portenta H7 functions as a USB host. This means that the Portenta H7 can receive and manage the information provided by other USB peripherals connected to it through the hub. In this scenario, the Portenta H7 is referred as "USB host" or "master" device, and the peripheral is known as the "client" device, ( usually a mouse or a keyboard ). Computers are by default USB hosts too, as they receive the information that USB peripherals provide. PCs follows a “plug and play” mechanism, this means that whenever a USB device is connected, the Host detects the device and loads the appropriate drives for the device to establish a communication channel over the serial bus.Besides being a host point for peripherals, Portenta H7 can also be connected to the computer as the "client". In this case, the PC is the USB host and Portenta H7 the peripheral. We see this scenario whenever we upload sketches to the board or when we monitor data through the serial monitor.Furthermore, the Portenta H7 can also be set to be a mouse or keyboard itself. With the corresponding sketch, circuitry and components, we could send keyboard or mouse data to a PC, but this would be a different project.
```

Comment:
The Portenta H7 can switch between a USB host and device based on our preferrence.